app/components/QuestionForm/index.js

```diff
@@ -147,120 +145,133 @@ export const QuestionForm = ({
+                      />
+                      <br />
+                    </>
+                  ))}
+
+              <TitleForm intl={intl} questionLoading={questionLoading} />
+
+              {formValues[FORM_TITLE] &&
+                formValues[FORM_TITLE].length >= 3 &&
+                (existingQuestions?.length ?? 0) > 0 &&
+                !doSkipExistingQuestions && (
+                  <ExistingQuestions
+                    questions={existingQuestions}
+                    skip={skipExistingQuestions}
+                    show={showMoreQuestions}
+                    intl={intl}
+                    communities={communities}
+                  />
+                )}
+
+              <ContentForm
+                intl={intl}
+                questionLoading={questionLoading}
                 formValues={formValues}
-                redirectToCreateTagDispatch={redirectToCreateTagDispatch}
               />
-            )}
-
-            {/*<BountyForm*/}
-            {/*  intl={intl}*/}
-            {/*  questionLoading={questionLoading}*/}
-            {/*  formValues={formValues}*/}
-            {/*  change={change}*/}
-            {/*  dotRestriction={DEFAULT_DOT_RESTRICTION}*/}
-            {/*/>*/}
-
-            {/*<BountyDateForm*/}
-            {/*  intl={intl}*/}
-            {/*  questionLoading={questionLoading}*/}
-            {/*  formValues={formValues}*/}
-            {/*  change={change}*/}
-            {/*/>*/}
-
-            <Button
-              disabled={questionLoading}
-              id={submitButtonId}
-              type="submit"
-            >
-              {submitButtonName}
-            </Button>
-          </FormBox>
-        </BaseSpecialOne>
 
-        <Tips />
-      </TipsBase>
-    </div>
+              <TagsForm
+                intl={intl}
+                questionLoading={questionLoading}
+                formValues={formValues}
+                change={change}
+              />
+
+              {profileWithModeratorRights && (
+                <SuggestTag
+                  formValues={formValues}
+                  redirectToCreateTagDispatch={redirectToCreateTagDispatch}
+                />
+              )}
+
+              {/*<BountyForm*/}
+              {/*  intl={intl}*/}
+              {/*  questionLoading={questionLoading}*/}
+              {/*  formValues={formValues}*/}
+              {/*  change={change}*/}
+              {/*  dotRestriction={DEFAULT_DOT_RESTRICTION}*/}
+              {/*/>*/}
+
+              {/*<BountyDateForm*/}
+              {/*  intl={intl}*/}
+              {/*  questionLoading={questionLoading}*/}
+              {/*  formValues={formValues}*/}
+              {/*  change={change}*/}
+              {/*/>*/}
+
+              <Button
+                disabled={questionLoading}
+                id={submitButtonId}
+                type="submit"
+                onClick={() => {
+                  if (isEdited) setIsNotPressed(!isNotPressed);
```

Comment:
Is there ever a case when the value will be changing from `true` to `false` and then back to `true`?
If not then this code is a bit confusing. `setIsNotPressed(false)` would be easier to understand.